content/docs/secure/encryption/troubleshooting.md

```diff
@@ -85,3 +85,18 @@ Resolution:
 - check the license is for the cluster on which the encrypted volumes are created
 - check [encryption-license secret](../deployment#secret-encryption-license)
 
+# Typical Rekey Failure reasons
+If all rekeys in the cluster are failing 
+- Check the Rekey controller helm chart values.yaml `provisioner` name against the encryption chart `pluginName`, and ensure they match.
+- Check the Rekey controller helm chart values.yaml `port` number against the encryption chart rekey port number, and ensure they match.

```

Comment:
Check the Rekey Controller Helm chart `port` value against related Dell CSI driver chart `encryption.apiPort` value.

Reply:
fixed.